assets/js/components/settings/SettingsCardConsentMode.js

```diff
@@ -47,10 +47,15 @@ const { useDispatch, useSelect } = Data;
 export default function SettingsCardConsentMode() {
 	const viewContext = useViewContext();
 
-	const isAdsConnected = useSelect( ( select ) =>
-		// TODO: Replace this with the `ads` version of the `getAdsConversionID()` selector once it's implemented.
-		select( MODULES_ANALYTICS_4 ).getAdsConversionID()
-	);
+	const isAdsConnected = useSelect( ( select ) => {
+		// TODO: Replace the `getAdsConversionID()` selector with its `ads`
+		// version once it's implemented.
+
+		const { getAdsConversionID, getAdsLinked } =
+			select( MODULES_ANALYTICS_4 );
+
+		return getAdsConversionID() || getAdsLinked();
+	} );
```

Comment:
This exact logic is used across three different components to do the same thing, let's create a new selector that returns a `boolean` from `!!getAdsConversionID() || !!getAdsLinked()` instead of doing this three different places 🙂

Reply:
As discussed, I've added a selector and re-used it in both places. Please let me know if this looks good now, thanks!